backend/app/models.py

```diff
@@ -133,3 +133,38 @@ def serialize_list(users):
 
     def __repr__(self):
         return "<User %r>" % self.email
+
+
+# Contact Model
+class Contact(db.Model):
+    __tablename__ = "contacts"
+    contact_id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4())
+    name = db.Column(db.String(256), nullable=False)
+    email = db.Column(db.String(256), nullable=False)
+    secondaryEmail = db.Column(db.String(256), nullable=False)
```

Comment:
I think secondary email can be empty, so remove the `nullable=False`. Also to follow Python naming conventions, it would be snake cased so change `secondaryEmail` to `secondary_email`